stable/grafana/values.yaml

```diff
@@ -154,6 +153,35 @@ persistence:
   # subPath: ""
   # existingClaim:
 
+initChownData:
+  ## If false, data ownership will not be reset at startup
+  ## This allows the prometheus-server to be run with an arbitrary user
+  ##
+  enabled: true
+
+  ## initChownData container name
+  ##
+  name: init-chown-data
+
+  ## initChownData container image
+  ##
+  image:
+    repository: busybox
+    tag: latest
+    pullPolicy: IfNotPresent
```

Comment:
i think we can remove this one if we pin to a specific image tag

Reply:
I think will be good idea leave this option for if someone wants to try with another image and need to download each time. What do you think? 

If we delete from the values.yaml we need to change the deployment.yaml to take in case of exist.

Reply:
i agree with that.